zgrab_schema.py

```diff
@@ -670,6 +670,16 @@
 
 zschema.registry.register_schema("zgrab-s7", zgrab_s7)
 
+zgrab_smb = Record({
+    "data":SubRecord({
+        "smb":SubRecord({
+           "smbv1":Boolean(), 
```

Comment:
i might label that last field `smb1_support` to make it clear that it's not a subrecord.